### PR TITLE
Improve code coverage for UI

### PR DESCRIPTION
**Pull request by ap891843**
Improve code coverage for UI

### CODE REVIEW

**Review by temanbrcom — CHANGES_REQUESTED** [review]
Missing "test:" tag on commit messages

**Comment by temanbrcom** [comment]
@asatklichov 
@grianbrcom 
@Nurkambay

**Comment by ap891843** [comment]
> Missing "test:" tag on commit messages

Done

File: clients/cobol-lsp-vscode-extension/jest.config.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ module.exports = {
       "<rootDir>/dist",
     ],
     "collectCoverage": true,
+    "coveragePathIgnorePatterns":["<rootDir>/src/_tests_/*.ts"],
```
Comment (asatklichov) [inline clients/cobol-lsp-vscode-extension/jest.config.js:13]: Have you tested in SonarCloud if this folder really excluded?

Comment (ap891843) [reply clients/cobol-lsp-vscode-extension/jest.config.js:13]: Looked into it now. It doesn't exclude.

**Comment by ap891843** [comment]
Sonar test coverage report:
image

**Review by asatklichov — APPROVED** [review]
👍

**Comment by temanbrcom** [comment]
Could you, please, squash the duplicated commits

**Pull request closed without merging**